### PR TITLE
Intro Page Upload

### PR DESCRIPTION
**Pull request by amyleblond**
changes:
**application.html** - added indicator of what can be changed (front matter) when logged in as AC
**intro.html** - delete html, add upload  txt reference, add indicator of what can be change (intro_page) when logged in as AC, also the home and logout buttons will show on this page which makes sense as AC
**upload_front_matter.html**: added the intro page method to upload, rearranged in a table
**views.py**: add function for writing the new file, calling it from its path and addition to intro page

for testing: collectstatic :) 
screen shot 2018-03-05 at 12 40 55 pm
screen shot 2018-03-05 at 12 41 03 pm
screen shot 2018-03-05 at 12 41 19 pm
screen shot 2018-03-05 at 12 41 45 pm

### CODE REVIEW

File: Documentation/README.md [hunk 1]
```diff
@@ -11,7 +11,22 @@
...
 0. Login to the Application - (https://team4.uwaterloo.ca/login/)
-1. [Review Applicants]
+1. [Change Application Form]
+	* Function: This page allows the AC to change the text on two pages, the Application Intro Page (https://team4.uwaterloo.ca/taform/) and the Application Page (https://team4.uwaterloo.ca/taform/application.html).
+	* Scenarios for Use:
+		* 1 - Application guidelines have changed.
+		* 2 - The term needs to be changed.
+		* 3 - A reference link has become outdated.
+	* How to Use:
+		* Login to the Application - (https://team4.uwaterloo.ca/login/)
```
Comment (j2kan) [inline Documentation/README.md]: remove.

they should be logged in from step 0.

File: Documentation/README.md [hunk 2]
```diff
@@ -11,7 +11,22 @@
...
+		* Open Step 4, Change Application Form - (https://team4.uwaterloo.ca/taform/upload_front_matter.html)
+		* For each file you may change (Intro Page, Front Matter), review the current page.
+		* Download the file you would like to change.
+		* Make changes within the downloaded file. Ensure you are using HTML syntax and save the file as .txt.
+		* Select the file you would like to upload.
+		* Upload the file.
+		* Review the changes by previewing the page you changed again.
```
Comment (j2kan) [inline Documentation/README.md]: I would add:

to test their page locally.

They can re save their txt file with the html extension and open it in the browser. (from ada's process)
But must upload with txt extension.

File: taworks/taform/templates/taform/upload_front_matter.html [hunk 3]
```diff
@@ -3,26 +3,62 @@
...
-<h1>Upload Front Matter</h1>
+<h1>Change Application Form</h1>
 
 <div class="successMessage">{{success}}</div>
 <div id="font">{{error}}</div>
-<p> Please ensure the file you upload is a text file (only .txt files will be uploaded). This file should contain HTML syntax. </p>
-<p> Click <a href="../../static/taform/front_matter.txt" download="front_matter.txt">here</a> for the current front matter. </p>
+<p>Please ensure the files you upload are text files (only .txt files will be uploaded). These files should contain HTML syntax. </p>
+
 <p id="p01"> Please note any uploads will replace current text file.</p>
```
Comment (j2kan) [inline taworks/taform/templates/taform/upload_front_matter.html]: i don't think p01 exist anymore, remove?

**Review by j2kan — APPROVED** [review]
left a few readMe comments otherwise LGTM

**Review by lauradobson — APPROVED** [review]

**Pull request merged**